### PR TITLE
Add TomatoBar.app v3.3.0

### PR DESCRIPTION
**Pull request by cayacdev**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/tomatobar.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "tomatobar" do
+  version "3.3.0"
+  sha256 "ff4d465f0d70cb2e234918a650d3844352d0179e9f3742666ae98d909988dc23"
+
+  url "https://github.com/ivoronin/TomatoBar/releases/download/v#{version}/TomatoBar-v#{version}.zip"
+  name "tomato-bar"
+  desc "🍅 World's neatest Pomodoro timer for menu bar"
```
Comment (bevanjkay) [inline Casks/tomatobar.rb:7]: From the Cask cookbook - [Do not use empty marketing jargon.](https://docs.brew.sh/Cask-Cookbook#stanza-desc)
```suggestion
  desc "Menu bar pomodoro timer"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by cayacdev** [comment]
> Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

Thank you for your review. I added a zap stanza and applied your suggestion for the description.

File: Casks/tomatobar.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask "tomatobar" do
+  version "3.3.0"
+  sha256 "ff4d465f0d70cb2e234918a650d3844352d0179e9f3742666ae98d909988dc23"
+
+  url "https://github.com/ivoronin/TomatoBar/releases/download/v#{version}/TomatoBar-v#{version}.zip"
+  name "tomato-bar"
```
Comment (bevanjkay) [inline Casks/tomatobar.rb:6]:
```suggestion
  name "TomatoBar"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/tomatobar.rb (hunk 2 above)
Comment (bevanjkay) [reply Casks/tomatobar.rb:6]: @cayacdev After this last change it should be good to go.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**